tools/dev/requirements.txt

```diff
@@ -0,0 +1,19 @@
+# Remote-pdb can be used inside build jobs, or parallelized aspects
+remote-pdb
+
+# Add temporary dev requirements with `-e` here
+#
+# Note: This should always be empty on `main`.
```

Comment:
Add a CI check for this?

Reply:
i thought about this, its not trivial to do without making this useless for ci

ie ci fails the deps check as it has a dev requirement, but then doesnt use the dev requirement that you want to test

Reply:
Different CI stages?

Reply:
that works as long as the its not used by any later stage - currently the check would go in the dep check which would prevent anything after the first stage

i can add it, but on balance i would prefer to keep this as a visual inspection in PRs where it has been (temporarily added)

Reply:
OK, I don't feel that strongly, just generally prefer machines do the work.